src/components/Property/RaiseWorkOrder/RateScheduleItemView.js

```diff
@@ -70,7 +70,7 @@ const RateScheduleItemView = ({
   }
 
   const onRateScheduleItemSelect = (index, event) => {
-    document.getElementById('priorityDescription').disabled = false
+    document.getElementById('priorityCode').disabled = false
```

Comment:
This is a kind of global side effect of this component (this will error if reused in another context without a priority code somewhere on the page) but I wasn't sure of a solution at the moment. Ideally the parent would know about the SOR change from its own state and then set a flag on the priority select. Might be one for another time.